website/docs/r/api_key.html.markdown

```diff
@@ -42,7 +42,7 @@ In addition to all arguments above, the following attributes are exported:
 * `api_key_id` - Unique identifier for this Organization API key.
 ## Import
 
-API Keys must be imported using org ID, API Key ID e.g.
```

Comment:
It would appear this documentation is accurate: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/mongodbatlas/resource_mongodbatlas_api_key.go#L171. Was the intention to modify project_api_key docs?

Reply:
I think you're referring to api_key but the PR is about project_api_key, e.g. the code is:

```
func resourceMongoDBAtlasProjectAPIKeyImportState(ctx context.Context, d *schema.ResourceData, meta any) ([]*schema.ResourceData, error) {
...
	projectID := parts[0]
	apiKeyID := parts[1]

	projectAPIKeys, _, err := conn.ProjectAPIKeys.List(ctx, projectID, nil)
```

thanks for noticing that I changed the wrong file :-)